app/Console/Commands/Import/ImportScheduleDaily.php

```diff
@@ -28,6 +28,8 @@ public function handle()
             '--yes' => 'default',
             'endpoint' => 'event-occurrences'
         ]);
+
+        $this->call('import:analytics');
```

Comment:
It looks like `import:analytics` doesn't have `protected $isPartial = true;`, so the full set of analytics data would be imported daily. That might be too much to do on a daily basis, though given that gathering the analytics data is a long-running process, maybe it does make sense to run it daily as partial?

Reply:
I was planning on loading the last-three-months data into the data service every day. Once the data is in the data service, it's not a big load to get all the data. Each record is small and there's only 100,000. Running a partial might get about half of that daily? It that different worth running it partially?